packages/website/src/pages/packages/[name]/index.tsx

```diff
@@ -11,32 +12,21 @@ const NoSSR = dynamic(
   }
 );
 
-/*export async function generateMetadata({
-  params,
-}: {
-  params: { name: string; tag: string; variant: string };
-}) {
-  const metadata: Metadata = {
-    title: `Cannon | ${params.name}`,
-    description: 'Package',
-    openGraph: {
-      title: `Cannon | ${params.name}`,
-      description: 'Package',
-      url: 'https://usecannon.com',
-      siteName: 'Cannon',
-      locale: 'en_US',
-      type: 'website',
-      images: [
-        {
-          url: 'https://usecannon.com/images/og.png',
-        },
-      ],
-    },
-  };
-  return metadata;
-  }*/
-
 export default function Deployment() {
   const params = useRouter().query;
-  return <NoSSR name={params.name as string} />;
+  return (
+    <>
+      <NextSeo
+        {...defaultSEO}
+        title={`Cannon | ${params.name}`}
```

Comment:
can we do this same pattern for all of the pages under the `packages`? Right now a lot of the pages are just "Package" when we should probably be embedding the params name like we are here